browser/BrowserPerf.proto

```diff
@@ -45,15 +45,17 @@ message BrowserPerfData {
     // Page path in the browser, mostly it is URI, without parameter
     string pagePath = 4;
     int32 pagePathId = 5;
+    // Then the PV with error is only calculated when firstReportedError is true.
+    bool firstReportedError = 6;
```

Comment:
After release, such as 7.0.0, we only add field at the last seq for keeping the compatible.